libs/common/src/sql-db-entities/fee-record/fee-record.entity.ts

```diff
@@ -0,0 +1,118 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { UtilisationReportEntity } from '../utilisation-report';
+import { Currency } from '../../types';
+import { AuditableBaseEntity } from '../base-entities';
+import { CreateFeeRecordParams } from './fee-record.types';
+import { getDbAuditUpdatedByUserId } from '../helpers';
+import { MonetaryColumn, ExchangeRateColumn } from '../custom-columns';
+
+@Entity('FeeRecord')
+export class FeeRecordEntity extends AuditableBaseEntity {
+  @PrimaryGeneratedColumn()
+  id!: number;
+
+  /**
+   * The `_id` of the associated facility in the 'facilities' MongoDB collection
+   */
+  @Column()
+  facilityId!: string;
+
+  /**
+   * The associated report from the UtilisationReport table
+   */
+  @ManyToOne(() => UtilisationReportEntity, (report) => report.feeRecords, {
+    nullable: false,
+  })
+  report!: UtilisationReportEntity;
+
+  /**
+   * Name of the exporter that the GEF is on behalf of
+   */
+  @Column()
+  exporter!: string;
+
+  /**
+   * Currency of the facility that the fee record is for
+   */
+  @Column({ type: 'nvarchar' })
+  baseCurrency!: Currency;
+
+  /**
+   * The current utilisation (drawdown) of the facility by the exporter
+   */
+  @MonetaryColumn()
+  facilityUtilisation!: number;
```

Comment:
This is a different type to the diagram on the ticket but I'm guessing this is actually correct. I'm not sure how important the diagram is so just noting in case it is.

Reply:
Good catch! I have updated the diagram and will put that on the jira ticket